config/webpack.config.js

```diff
@@ -52,11 +65,22 @@ var config = {
       path.join(__dirname, '..', 'webpack'),
       'node_modules/'
     ],
-    alias: {
+    alias: Object.assign({
       foremanReact:
         path.join(__dirname,
-           '../webpack/assets/javascripts/react_app/components')
-    }
+           '../webpack/assets/javascripts/react_app/components'),
+      foremanReactCommon:
+        path.join(__dirname,
+          '../webpack/assets/javascripts/react_app/common'),
+      foremanReactService:
+        path.join(__dirname,
+          '../webpack/assets/javascripts/services'),
+      foremanNodeModules:
```

Comment:
This is here because of [packaging-related changes](https://github.com/theforeman/foreman/pull/4719), so that plugins know where the foreman's node_modules are.

```js
import React from 'foremanNodeModules/react';
```

Reply:
for example, if I import a 3rd party react component (lets say a date picker) then it code will include
```js
import React from 'react';
```

This will:
a. make us pull another copy of react into the plugin node modules
b. will include another copy of react in the plugin bundle.

Reply:
/cc @dLobatog - this is probably a side effect we did not expect.